src/api.js

```diff
@@ -0,0 +1,7 @@
+const URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=a2426096&t=';
+
+export function getMovie(title) {
+  return fetch(`${URL}${title}`)
+    .then(response => response.json())
+    .catch(er => 'error');
```

Comment:
if you want catch an error it's better to show it in proper way , not just returning a string, but wrap it in alert call or even better throw an custom Error object
https://learn.javascript.ru/try-catch#generatsiya-sobstvennyh-oshibok